docker-compose.yml

```diff
@@ -29,11 +28,10 @@ services:
       - .:/usr/src/app
       - gem_cache:/gems
       - ./example/datasets:/tmp/datasets
-    command: bundle exec sidekiq -r ./lib/sidekiq_jobs.rb
+    command: bundle exec sidekiq -c 1 -r ./lib/sidekiq_jobs.rb
```

Comment:
This ensures sidekiq uses only a single thread (no benefit to multiple threads, since MRI can only use one CPU core and our jobs tend to be CPU bound)